src/compiler/checker.ts

```diff
@@ -40150,7 +40442,7 @@ namespace ts {
                 // If we hit an export assignment in an illegal context, just bail out to avoid cascading errors.
                 return;
             }
-
+            // @ts-ignore

```

Comment:
Definitely can't have a `//@ts-ignore` for any reason in our codebase. There's always some other way to handle whatever's going on.